openedx/core/djangoapps/course_date_signals/waffle.py

```diff
@@ -0,0 +1,19 @@
+"""
+This module contains various configuration settings via waffle switches for
+course date signals.
+"""
+
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+WAFFLE_FLAG_NAMESPACE = "course_date_signals"
+
+# .. toggle_name: course_date_signals.disable_spaced_out_sections
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to disable spaced out sections for self paced courses.
```

Comment:
@Cup0fCoffee, nit: please add an explanation of what the "spaced out sections" are. This will be listed on [the Feature Toggles page](https://docs.openedx.org/projects/edx-platform/en/latest/references/featuretoggles.html), so it would be nice to understand what exactly it's doing without digging into the code.

Also, it might be good to give the flag a more verbose and intuitive name, like `course_date_signals.relative_dates_disable_suggested_schedule` or something.

Reply:
@Agrendalath I've added a commit that improved description. Could you have a quick look?

In regards to flag name, I agree that `spaced_out_sections` is not self explanatory, however I like that this "term" is quite specific and relatively concise, and once you know what it means, it's nice that it's consistent throughout the code base and docs, so I would prefer to keep it like this.

Reply:
FYI, the quality checks are failing on commit message linting, it will pass once I squash.

Reply:
@Cup0fCoffee,

> In regards to flag name, I agree that spaced_out_sections is not self explanatory, however I like that this "term" is quite specific and relatively concise, and once you know what it means, it's nice that it's consistent throughout the code base and docs, so I would prefer to keep it like this.

We should choose verbosity for feature flags to make them as intuitive as possible.

> it's nice that it's consistent throughout the code

The target audience of feature flags is usually site operators, not developers.

> and docs

Unfortunately, searching for "Open edX spaced out sections" doesn't return anything related to this.

Reply:
@Agrendalath Ok, I've changed the user facing name of the flag, updated the description to use "suggested relative due dates" instead of "spaced out", and made the description more verbose.